docs/components/servo/_index.md

```diff
@@ -339,6 +339,14 @@ You can find additional assistance in the [Troubleshooting section](/appendix/tr
 
 {{< snippet "social.md" >}}
 
+## Test the servo
+
+After setting up and establishing the connection for your servo, navigate to the [Control tab](https://docs.viam.com/manage/fleet/robots/#control) and select the dropdown panel dedicated to the motor.
+
+![A photo of the servo component in the control tab](assets/components/servo/servo-control-tab.png)
```

Comment:
```suggestion
![The servo component in the control tab](assets/components/servo/servo-control-tab.png)
```

The text here is alt text and when it's read by a screenreader most screenreaders will prepent something like picture of, which then results in it saying "picture of a photo of..." - so it's best to exclude "A photo of".